std/socket.d

```diff
@@ -1863,6 +1863,18 @@ public:
         sin6 = addr;
     }
 
+    version (Posix)
+    {
+        /// Human readable string representing the IPv6 address in RFC 2373 form.
+        override string toAddrString() @trusted const
+        {
+            char* buf = cast(char*) .malloc(INET6_ADDRSTRLEN);
+            string addrString = to!string(.inet_ntop(AddressFamily.INET6, &sin6.sin6_addr, buf, INET6_ADDRSTRLEN));
+            .free(buf);
```

Comment:
```suggestion
            char[INET6_ADDRSTRLEN] buf;
            string addrString = to!string(.inet_ntop(AddressFamily.INET6, &sin6.sin6_addr, buf.ptr, INET6_ADDRSTRLEN));
```
Since `INET6_ADDRSTRLEN` is a compile-time constant, we can use a static array here and avoid heap allocation.